firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -196,6 +196,30 @@ extension String {
                 value: "Delete Bookmark",
                 comment: "The title for the Delete Bookmark button, in the Edit Bookmark popup screen which is summoned from the main menu's Save submenu, which will delete the currently bookmarked site from the user's bookmarks.")
         }
+        public struct RootFolderEmptyState {
```

Comment:
I won't block the PR on this, but, we should think about organizing strings in logical, and shared structs. So, this could be Bookmarks.EmptyState.Root.Title. The way I try to think about string organization is this: if someone adds strings in the string file, and I now need to go and add those strings to the code, even thought I wasn't privy to the PR for the strings being added, how easily logical and discoverable is that string in both "guessing" terms, but also, in "autocomplete" terms. So, for example, if a struct has, say, 100 things in it... it's gonna be difficult to discover things in there because who wants to scroll through 100 autocomplete entries? Whereas it if it has, say, 5 major categories, than then get split, it's much easier to move through the code that way.